server/publicapi/packages/service.py

```diff
@@ -9,13 +9,13 @@
 # at https://www.sourcefabric.org/superdesk/license
 
 import logging
-from superdesk.services import BaseService
+from publicapi.items.service import ItemsService
 
 
 logger = logging.getLogger(__name__)
 
 
-class PackagesService(BaseService):
+class PackagesService(ItemsService):
```

Comment:
@petrjasek Isn't a package more of a _container_ of items? (at least by my understanding)
It's like saying that a box is a specific type of shoe, because it can contain different types of shoes and other boxes.

Please correct me if this analogy is not correct.